src/index.js

```diff
@@ -0,0 +1,7 @@
+exports.say = function (name) {
+    if(name!=null){
+        return('Hello '+name+'!');
```

Comment:
The braces in return statements are optional `return 'Hello ' + name + '!'`
You can also use ECMA6 templates with backticks. Something like:
`` return `Hello ${name}!` ``